samples/highcharts/demo/horizontal-organizational-chart/demo.details

```diff
@@ -0,0 +1,10 @@
+---
+name: Horizontal organizational chart
```

Comment:
The term we use consistently is "organization chart". So this should be reflected in both the name of the sample and in the file path.

Reply:
There is already a /demo/organization-chart existing

Reply:
But this one should be `/demo/horizontal-organization-chart`, right?

Reply:
Ahh, I misunderstood this! Sorry 😀